rpi_backlight/cli.py

```diff
@@ -51,19 +52,32 @@ def _create_argument_parser():
     parser.add_argument(
         "-B",
         "--board-type",
-        default="raspberry-pi",
+        default=_get_board(),
         choices=BOARD_TYPES.keys(),
         help="board type",
     )
     parser.add_argument(
-        "-V",
-        "--version",
-        action="version",
-        version=f"%(prog)s {__version__}",
+        "-V", "--version", action="version", version=f"%(prog)s {__version__}"
```

Comment:
It shouldn't if you keep the comma - once you remove it, black will attempt to put the multiple lines back into one :)

Reply:
No matter what I do it is reformatting this for some reason
Perhaps because its short enough to fit on one line?
I am entering 
```
    parser.add_argument(
        "-V",
        "--version",
        action="version",
        version=f"%(prog)s {__version__}",
    )    
    return parser
```

and it's getting blacked to one line
I am using the latest version of black

Reply:
Actually, it did it to your --duration too, in your own repo
https://github.com/linusg/rpi-backlight/blob/main/rpi_backlight/cli.py#L49

Reply:
No, these have been like that for 2+ years...

>  I am using the latest version of black availabile to me on apt

try installing it via pip then, this is quite old - almost three years! latest black is `21.6b0`. I'd generally not recommend installing Python packages via apt.

Reply:
Installing via pip did the trick, thanks
```
root@Harbormaster:~# black --version
black, version 21.6b0
```

Reply:
Please add the comma and restore the original formatting :)

Reply:
Done, sorry!